packages/video_player/ios/Classes/VideoPlayerPlugin.m

```diff
@@ -335,6 +331,17 @@ - (int64_t)position {
 }
 
 - (int64_t)duration {
+  if (CMTIME_IS_INDEFINITE([[_player currentItem] duration])) {
```

Comment:
I assume this is computing the length of loaded times in livestream. I personally would think the duration is the length of the whole video. Maybe create a different method for getting the "loadedTimes" for livestream videos?